alerts/google-quotas/metadata.yaml

```diff
@@ -24,16 +24,19 @@ alert_policy_templates:
   description: "This policy will notify you if quota usage reaches the alert trigger threshold of 80%. Expand the drop-down below if you would like to edit the alert trigger threshold or further customize this alert policy."
   version: 1
 -
-  id: all-quota-adjuster-changes
-  description: "This policy will notify you of any changes made by Quota Adjuster"
+  id: all-adjustements-by-quota-adjuster
```

Comment:
are the json file names updated to match these changes too?

Reply:
Yes the json file names are updated to match these changes